_data/authors.yml

```diff
@@ -167,3 +167,20 @@ Dave Placeholder:
       - label: "Blogsite"
         icon : "fas fa-fw fa-link"
         url  : "http://monsieurfromageblog.com"      # Author details.
+
```

Comment:
Hi Arjun - the authors is only used when you are contributing new content that you own.  We'll add you in anyway in case you create something in the future!!

Your changes will be recognised in the azurecitadel organisation's repo. 😀